Neos.ContentRepository/Classes/TypeConverter/NodeConverter.php

```diff
@@ -45,6 +45,11 @@ class NodeConverter extends AbstractTypeConverter
      */
     const REMOVED_CONTENT_SHOWN = 1;
 
+    /**
+     * @var boolean
```

Comment:
Seems not to be a boolean :)
```suggestion
     * @var int
```

Reply:
Sorry, seems I never been in that part of the code :/
So thats may be dumb question: Why do we have integer keys here, and not something more meaningful?

Reply:
Mainly because it doesn't matter–the constant name is the meaningful part.